src/libs/actions/ReportActions.ts

```diff
@@ -33,14 +35,55 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction) {
         return;
     }
 
+    if (keys) {
+        const errors: Record<string, null> = {};
+
+        keys.forEach((key) => {
+            errors[key] = null;
+        });
+
+        Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`, {
+            [reportAction.reportActionID]: {
+                errors,
+            },
+        });
+        return;
+    }
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`, {
         [reportAction.reportActionID]: {
             errors: null,
         },
     });
 }
 
-export {
-    // eslint-disable-next-line import/prefer-default-export
-    clearReportActionErrors,
-};
+/**
+ * 
+ignore: undefined means we want to check both parent and children report actions
+ignore: parent or child means we want to ignore check parent or child report actions because we already check them before
+ */
```

Comment:
```suggestion
/**
 * ignore: `undefined` means we want to check both parent and children report actions
 * ignore: `parent` or `child` means we want to ignore checking parent or child report actions because they've been previously checked
 */
```